ontology/slso.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: ontology_detail
+id: slso
+title: Space Life Sciences Ontology
+browsers:
+- title: BioPortal Ontology Browser
+  label: BioPortal
+  url: https://bioportal.bioontology.org/ontologies/SLSO
+contact:
+  email: daniel.c.berrios@nasa.gov
+  github: DanBerrios
+  label: Dan Berrios
+  orcid: 0000-0003-4312-9552
+description: The Space Life Sciences Ontology is an application ontology and is intended to support the operation of NASA's Life Sciences Data Archive and other systems that contain space life science research data.
+domain: investigations
+homepage: https://github.com/nasa/LSDAO
+in_foundry_order: 1
```

Comment:
```suggestion
```

Reply:
Why this statement?

Reply:
> Hmm. I bet @DanBerrios copied this from another config, like OBI's. It should be removed. Thanks @matentzn.

Yeah I wasn't sure what it meant and copied it from the PATO example...